modules/configure-cvo-retrieve-image-signatures.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+// * updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configure-cvo-retrieve-image-signatures_{context}"]
+= Configuring the Cluster Version Operator (CVO) to retrieve release image signatures
+
+If you are using the oc-mirror plugin 4.15 or later and the OSUS Operator 5.0.3 or later to update your cluster in a disconnected environment, you can configure the Cluster Version Operator (CVO) to retrieve release image signatures from an endpoint on the OpenShift Update Service (OSUS) in your environment.
+
+You can configure this by adding the URI of OSUS' signature serving endpoint to the CVO's list of signature stores through the `signatureStores` property.
+This is a one-time configuration that avoids the need to apply release image signatures each time you mirror a new set of release images.
+
+:FeatureName: The `signatureStores` property
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* You have deployed OSUS using the OSUS Operator 5.0.3 or later.
+* You are using the oc mirror plugin 4.15 or later to mirror release images, and have set `graph:true` in the image set configuration.
+* You have enabled the `TechPreviewNoUpgrade` feature set in your cluster.
+
+.Procedure
+
+. Retrieve the signature serving URI of OSUS by running the following command:
++
+[source,terminal]
+----
+$ while sleep 1; do SIGNATURES_URI="$(oc -n "${NAMESPACE}" get -o jsonpath='{.status.metadataURI}/api/upgrades_info/signatures{"\n"}' updateservice "${NAME}")"; SCHEME="${SIGNATURES_URI%%:*}"; if test "${SCHEME}" = http -o "${SCHEME}" = https; then break; fi; done
+----
+
+. Patch the cluster version resource by running the following command:
++
+[source,terminal]
+----
+$ oc patch clusterversion version --type=merge -p '{"spec":{"signatureStores":[{"url": "'${SIGNATURES_URI}'"}]}}'
+----
```

Comment:
Because the signaturestore provided by osus is behind self-signed ingress CA, if the ca is not enabled in cluster-wide proxy, CVO provides a config through above api to setting it explicitly. Following is an example steps to add the ca cm.
 
```
# ./oc -n openshift-config create configmap valid-ca --from-file=ca.crt
configmap/valid-ca created

# ./oc patch clusterversion version --type merge -p '{"spec": {"signatureStores": [{"url": "https://update-service-oc-mirror-meta-route-openshift-update-service.apps.jliu416d.qe.gcp.devcluster.openshift.com/api/upgrades_info/signatures","ca": {"name": "valid-ca"}}]}}'
clusterversion.config.openshift.io/version patched
```
I'd suggest to add the note here as a plus configure step.

Reply:
new update about the ca support of signaturestore:
the pr for https://issues.redhat.com/browse/OCPBUGS-24700 has been reverted due to regression hours ago, i guess the ca part which implemented in the bug will miss the scope of 4.16 GA, so maybe we could ignore above comment to mention the set of `ca`. 
 cc @wking @PratikMahajan

Reply:
Is the rest of the content in this section still at risk of not making it to 4.16? I can still proceed with QE review and peer review just in case, but I'm curious if there are any updates on the state of the features this PR is documenting